### PR TITLE
Fix incorrect public beta bindings example

### PR DESCRIPTION
**Pull request by vlovich**

### CODE REVIEW

File: content/r2/get-started.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ async function handleRequest(request) {
       await MY_BUCKET.put(key, request.body);
       return new Response(`Put ${key} successfully!`);
    case 'GET':
-     const { value } = await MY_BUCKET.get(key);
+     const { body } = await MY_BUCKET.get(key);
```
Comment (WalshyDev) [inline content/r2/get-started.md]: the object is nullable no? So this will error if the key doesn't exist

File: content/r2/get-started.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ async function handleRequest(request) {
...
 
      if (value === null) {
```
Comment (WalshyDev) [inline content/r2/get-started.md]: needs to be updated along with line 161

**Review by WalshyDev — COMMENTED** [review]
@Cherry did a PR here: https://github.com/cloudflare/cloudflare-docs/pull/4262

Just needs merging. For this PR other lines need updating:

File: content/r2/get-started.md (hunk 1 above)
Comment (vlovich) [reply content/r2/get-started.md]: If the key doesn't exist then `.get` throws an exception

**Comment by vlovich** [comment]
Superseded by https://github.com/cloudflare/cloudflare-docs/pull/4262

**Pull request closed without merging**